File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex16.java
//Importando Scanner
import java.util.Scanner;

public class Ex16{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	//vars
	double catetoAdjacente, catetoOposto, hipotenusa;
	
	System.out.println("Hipotenusa de um triângulo\n\n");
	
	//	Entradas
	System.out.print("Insira o tamanho do cateto adjacente: ");
	catetoAdjacente = input.nextDouble();
	
	System.out.print("Insira o tamanho do cateto oposto: ");
	catetoOposto = input.nextDouble();
	
	//Processamento
	hipotenusa = Math.sqrt( Math.pow(catetoAdjacente, 2) + Math.pow(catetoOposto, 2) );
	
	//Saída
	System.out.printf("Hipotenusa = %.2f", hipotenusa);
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex16.c
#include <stdio.h>
#include <math.h>

main()
{
    //vars
    float catetoOposto, catetoAdjacente, hipotenusa;

    printf("\t\t\tHipotenusa de um triangulo\n\n");

    //Entrada
    printf("Insira o tamanho do cateto adjacente: ");
    scanf("%f",&catetoAdjacente);

    printf("Insira o tamanho do cateto oposto: ");
    scanf("%f",&catetoOposto);

    //Processamento
    hipotenusa = sqrt( pow(catetoAdjacente, 2) + pow(catetoOposto, 2) );

    //Saída
    printf("Hipotenusa = %.2f", hipotenusa);

}
